server/src/service/DfspNetworkConfigService.js

```diff
@@ -166,19 +179,138 @@ const createDFSPIp = async (ctx, dfspId, body, direction) => {
+ *
+ * @param {String} dfspId DFSP id
+ * @returns BaseEndpoint + DFSPEgress
+ **/
+ exports.getDFSPEgress = async (ctx, dfspId) => {
+  await PkiService.validateDfsp(ctx, dfspId);
+  const result = await DFSPEndpointModel.findLastestObjectByDirection(dfspId, DirectionEnum.EGRESS);
+  if (result == null) throw new NotFoundError('Endpoint configuration not found!');
+  return transformEndpointModeltoApiRes(result);
+
+  // ## Example response
+  // return {
+  //   "id": 0,
+  //   "type": "IP",
+  //   "state": "NEW",
+  //   "createdDate": "2022-03-24T09:41:21.263Z",
+  //   "createdBy": "string",
+  //   "dfspId": "string",
+  //   "IPList": [
+  //     {
+  //       "description": "Notification Callback Egress IP & Ports",
+  //       "address": "163.10.24.28/30",
+  //       "ports": [
+  //         "80",
+  //         "8000-8080"
+  //       ]
+  //     }
+  //   ]
+  // };
+};
+
 /**
  * Adds a new IP entry to the DFSP Egress endpoint
  *
  * @param {String} dfspId DFSP id
- * @param body {Object} InputIP DFSP egress IP
- * returns DFSPEndPointIp
+ * @param body {Object} DFSPIngress
+ * @returns BaseEndpoint + DFSPIngress
  **/
 exports.createDFSPEgressIp = async (ctx, dfspId, body) => {
   const endpointItem = await createDFSPIp(ctx, dfspId, body, 'EGRESS');
   const id = await DFSPEndpointItemModel.create(endpointItem);
   return DFSPEndpointItemModel.findObjectById(id);
 };
 
+/**
+ * Adds a new configuration to DFSP Endpoint Ingress
+ *
+ * @param {String} dfspId DFSP id
+ * @param body {Object} DFSPIngress
+ * @returns BaseEndpoint + DFSPIngress
+ **/
+ exports.createDFSPIngress = async (ctx, dfspId, body) => {
+  await PkiService.validateDfsp(ctx, dfspId);
+  const resultId = await DFSPEndpointModel.create(dfspId, StatusEnum.NOT_STARTED, null, DirectionEnum.INGRESS, body);
+  return transformEndpointModeltoApiRes(await DFSPEndpointModel.findObjectById(resultId));
+
+  // ## Example response
+  // return {
+  //   "id": 0,
+  //   "type": "IP",
+  //   "state": "NEW",
+  //   "createdDate": "2022-03-24T09:47:48.477Z",
+  //   "createdBy": "string",
+  //   "dfspId": "string",
+  //   "value": {
+  //     "url": "string"
+  //   }
+  // };
+};
+
+/**
+ * Get newc configuration to DFSP Endpoint Ingress
+ *
+ * @param {String} dfspId DFSP id
+ * body InputIP DFSP ingress IP
+ * returns DFSPIngress
+ **/
+ exports.getDFSPIngress = async (ctx, dfspId) => {
+  await PkiService.validateDfsp(ctx, dfspId);
+  const result = await DFSPEndpointModel.findLastestObjectByDirection(dfspId, DirectionEnum.INGRESS);
+  if (result == null) throw new NotFoundError('Endpoint configuration not found!');
+  return transformEndpointModeltoApiRes(result);
+
+  // ## Example response
+  // return {
+  //   "id": 0,
+  //   "type": "IP",
+  //   "state": "NEW",
+  //   "createdDate": "2022-03-24T09:47:48.477Z",
+  //   "createdBy": "string",
+  //   "dfspId": "string",
+  //   "value": {
+  //     "url": "string"
+  //   }
+  // };
```

Comment:
I think it worth adding that to the openAPI schema, rather than placing it here:
https://swagger.io/docs/specification/adding-examples/